openedx/core/djangoapps/credit/models.py

```diff
@@ -392,6 +396,13 @@ def get_course_requirement(cls, course_key, namespace, name):
             return None
 
 
+@receiver(models.signals.post_save, sender=CreditRequirement)
+@receiver(models.signals.post_delete, sender=CreditRequirement)
+def invalidate_credit_requirement_cache(sender, **kwargs):   # pylint: disable=unused-argument
+    """Invalidate the cache of credit requirements. """
+    RequestCache.clear_request_cache(name=CreditRequirement.CACHE_NAMESPACE)
```

Comment:
Similar Q as above - would it be worthwhile to have a different method that only clears the cache entry for a specific course, rather than dropping the entire namespace? Or rather, can we include a `course_id` in the namespace to facilitate only-updating-the-course-that-changed?